test/integration/features/basic.feature

```diff
@@ -28,4 +36,4 @@ Feature: Basic
      When executing "minishift stop"
      Then Minishift should have state "Stopped"
      When executing "minishift delete"
-     Then Minishift should have state "Does Not Exist"
+     Then Minishift should have state "Does Not Exist"
```

Comment:
@gbraad It's not the new line it just that I put my features bit before it so that's comes it like that. I don't like this way so showing stuff from github (it's confusing) :(